src/control/SConscript

```diff
@@ -102,7 +99,37 @@ def scons():
                 join(denv.subst("$SPDK_PREFIX"), "include"),
                 Copy("$TARGET", "$SOURCE"))
 
-    show_go_info()
+    def CheckGoVersion(context):
+        context.Display('Checking for Go compiler in $PATH... ')
+        go_path = context.env.WhereIs(GO_COMPILER)
+        if go_path:
+            context.Display(go_path + '\n')
+        else:
+            context.Result(0)
+            return 0
+
+        context.Display('Checking %s version... ' % go_path)
+        out = os.popen('%s version' % go_path).read()
+        if len(out.split(' ')) < 3:
+            context.Result('failed to get version from "%s"' % out)
+            return 0
+
+        # go version go1.2.3 linux/amd64
+        go_version = out.split(' ')[2].replace('go', '')
+        if len([x for x, y in
+                zip(go_version.split('.'), MIN_GO_VERSION.split('.'))
+                if int(x) < int(y)]) > 0:
+            context.Result('%s is too old (min supported: %s) ' % (go_version, MIN_GO_VERSION))
+            return 0
+        else:
+            context.Result('%s' % go_version)
+            return 1
+
+    conf = Configure(denv, custom_tests={'CheckGoVersion': CheckGoVersion})
+    if not conf.CheckGoVersion():
+        print 'no usable Go compiler found (yum install golang?)'
```

Comment:
Do you want to provide install hints for the multiple platforms (or at least the platforms) we are targeting.  For SLES, (12.3 at least -- I have not done 15.1 yet) this is `zypper install go1.10` for example.

Reply:
I'd prefer not to go down that road unless people feel strongly that it's a requirement. As a developer, I would read that hint as "or the moral equivalent for your platform".